regression-test/suites/tpch_sf1/script/README

```diff
@@ -0,0 +1,19 @@
+# Manual
+
```

Comment:
We need a more detailed README. And make this script a little more general for any other case